crates/holochain/src/core/workflow/app_validation_workflow.rs

```diff
@@ -156,16 +130,19 @@ async fn app_validation_workflow_inner(
                 });
 
                 // Validate this op
+                let dht_op_hash = op.to_hash();
```

Comment:
for the map of missing dependency to DhtOp, the DhtOpHash is now passed all the way down to `run_validate_callback`.